### DISCUSSION TITLE
i am trying to change the pallettes

### DISCUSSION THREAD

**Discussion by Raptorq1** [Techokami/SonicWorldsNext#72] [Q&A] [answered]
I am trying to change the palettes but I don't understand it

**Comment by Renhoex** [comment, marked as answer]
if you're looking at the palette image, the first row of palettes is the base colours of the player with every other row being what the colours should be replaced with.
Only warning is that your palette image size needs to be to the power of 2.

If you still don't understand you're going to need to provide more details on what you're struggling with.
